build-image/Dockerfile

```diff
@@ -5,7 +5,11 @@ RUN apt-get update && apt-get install -y curl python-requests python-yaml file j
 	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt-get install -y nodejs npm && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-RUN npm install -g postcss-cli autoprefixer
+
+# Install website builder dependencies. Whenever you change these version, please also change website/package.json
+# and viceversa.
+RUN npm install -g postcss-cli@5.0.1 autoprefixer@9.8.5
```

Comment:
You could may be have added the package.json inside the image. Not sure.

Reply:
I tried. The problem is that dependencies from `package.json` cannot be installed globally (`npm install -g` doesn't read `package.json` or the lock), so I'm not sure how to achieve it. Do you have any idea?